src/tezos/TezosTools.ts

```diff
@@ -1,3 +1,5 @@
+import { InMemorySigner } from '@taquito/signer'
+import { validateAddress, ValidationResult } from '@taquito/utils'
```

Comment:
Since we use '@taquito/utils' directly, we should probably include it in package.json.